buf/extensions.bzl

```diff
@@ -16,11 +16,11 @@
 See https://bazel.build/docs/bzlmod#extension-definition
 """
 
-load("//buf/internal:toolchain.bzl", "buf_download_releases")
 load("//buf/internal:repo.bzl", "buf_dependencies")
+load("//buf/internal:toolchain.bzl", "buf_download_releases")
 
-_DEFAULT_VERSION = "v1.28.1"
-_DEFAULT_SHA256 = "05dfb45d2330559d258e1230f5a25e154f0a328afda2a434348b5ba4c124ece7"
```

Comment:
What should this SHA be? It doesn't seem like it can really be used...since it's gonna be dependent on the platform we end up downloading it from...should we just remove this?

Reply:
It is not platform dependent, it is the SHA of the file that contains the platform dependent hashes